### DISCUSSION TITLE
Is it possible to bind "gg"?

### DISCUSSION THREAD

**Discussion by jordan-bravo** [zellij-org/zellij#3155] [Q&A]
In programs such as vim and less, the keybinding for scrolling to the top is `gg` (pressing `g` twice).  Is there any way to achieve this in Zellij's scroll mode?  I tried the following:
```
keybinds {
    shared_among "search" "scroll" {
        bind "gg" {
            ScrollToTop
        }
    }
}
```

But it led to this error:
```
  × Failed to parse Zellij configuration
    ╭─[/home/jordan/.config/zellij/config.kdl:16:1]
 16 │             }
 17 │ ╭─▶         bind "gg" {
 18 │ │               ScrollToTop
 19 │ ├─▶         }
    · ╰──── Invalid key: 'gg'
 20 │         }
    ╰────
```

**Comment by plejd-sebman** [comment]
As I understand user defined mappings have to use CTRL + "some keys" to be clearly separated from built-in mappings. But I might be wrong.
